packages/agent/src/dwn-manager.ts

```diff
@@ -242,17 +242,18 @@ export class DwnManager {
   }
 
   private async constructDwnMessage(options: {
-    request: ProcessDwnRequest
+    request : ProcessDwnRequest
```

Comment:
Unnecessary space added in this PR that should be removed.
```suggestion
    request: ProcessDwnRequest
```

Reply:
Fixed